examples/Tilt-Compensated Compass Example

```diff
@@ -0,0 +1,77 @@
+## Tilt Compensated Compass using Accelerometer and Magnetometer
+
+# # Equations from st manual, "Computing tilt measurement and tilt-compensated eCompass" by Andrea Vitali
+   # https://www.st.com/resource/en/design_tip/dm00269987-computing-tilt-measurement-and-tiltcompensated-ecompass-stmicroelectronics.pdf
+    
+   # for Phi
+      #  Roll: phi = atan2(Gy, Gz) # for stability ,can be substituted with Gz+Gx*alpha (with alpha=0.01-0.05)
+   # for Theta
+      # Gz2 = Gy * Sin( Phi ) + Gz * Cos( Phi )
+      # Pitch: Theta = Atan( -Gx / Gz2)
+   # for Psi or "Yaw" (The Tilt Compensated Heading); Bx,By,Bz are magnetometer readings
+      # By2 = Bz * Sin( Phi ) – By * Cos( Phi )
+      # Bz2 = By * Sin( Phi ) + Bz * Cos( Phi )
+      # Bx3 = Bx * Cos( Theta ) + Bz2 * Sin( Theta )
+      # Yaw: Psi = Atan2( By2 , Bx3)
+###############################################
+
+import board
+import busio
+from digitalio import DigitalInOut
+import re
+import time
+from math import atan, atan2, degrees, pi, cos, sin
+import adafruit_lsm303_accel
+import adafruit_lis2mdl
+import adafruit_lsm303dlh_mag
+
+# Mag and Accel offsets from calibration. You will need to calibrate for your own offsets
+offmag = (-10.8, 4.875, -54.225)
+offaccel = (-0.994392, -0.305969, 0.382458)
+
+
+i2c = busio.I2C(board.SCL, board.SDA)
```

Comment:
We are no longer recommending this way of initiate the bus, we recommend board.I2C() and we could add a note 

```python
i2c = board.I2C()  # uses board.SCL and board.SDA
```

Reply:
Added changes and comment to clarify usage